README.md

```diff
@@ -0,0 +1,6 @@
+# Supergames
+supergames games developer
+
+hello this is supergames
+we are so happy that you chosse our games to run on your divice
+if youfind a bug please report it to us
```

Comment:
this is were the bugs go
[supergames issues](https://github.com/SuperGamesdev/Supergames/issues)